adaptors/hybriswakeupadaptor/hybriswakeupadaptor.h

```diff
@@ -0,0 +1,58 @@
+/****************************************************************************
+**
+** Copyright (c) 2025 Jollyboys Ltd.
+**
+** $QT_BEGIN_LICENSE:LGPL$
+**
+** GNU Lesser General Public License Usage
+** Alternatively, this file may be used under the terms of the GNU Lesser
+** General Public License version 2.1 as published by the Free Software
+** Foundation and appearing in the file LICENSE.LGPL included in the
+** packaging of this file.  Please review the following information to
+** ensure the GNU Lesser General Public License version 2.1 requirements
+** will be met: http://www.gnu.org/licenses/old-licenses/lgpl-2.1.html.
+**
+** $QT_END_LICENSE$
+**
+****************************************************************************/
+
+#ifndef HYBRISWAKEUPADAPTOR_H
+#define HYBRISWAKEUPADAPTOR_H
+#include "hybrisadaptor.h"
+
+#include <QString>
+#include <QStringList>
+#include <linux/input.h>
+#include "datatypes/timedunsigned.h"
+#include "deviceadaptorringbuffer.h"
+#include <QTime>
```

Comment:
Minor but I'd move this up together with other qt includes.

Reply:
moved